rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -119,3 +120,58 @@ TEST_F(RosBag2PlayTestFixture, recorded_messages_are_played_for_all_topics)
           &test_msgs::msg::Arrays::float32_values,
           ElementsAre(40.0f, 2.0f, 0.0f)))));
 }
+
+TEST_F(RosBag2PlayTestFixture, topic_qos_profiles_overriden)
+{
+  const auto topic1 = "topic1";
+  const auto topic2 = "topic2";
+  const auto msg_type = "test_msgs/BasicTypes";
+  auto basic_msg = get_messages_basic_types()[0];
+  basic_msg->int32_value = 42;
+
+  auto topic_types = std::vector<rosbag2_storage::TopicMetadata>{
+    {topic1, msg_type, "", ""},
+    {topic2, msg_type, "", ""},
```

Comment:
Please document what each field represents, as it's not really readable right now.